Introduce Workstream Participant Invited Expert Status (#148)
whatwg/sg: Workstream Participant Invitation Policy.md

(for example, the [Individual] is working in the [Field of Web Technologies] and the
[Entity] employing the individual is unable to join), may request special 
<a id="ws-invitation">**Workstream Participant Invitation**</a> status. This policy describes the conditions and
procedures for this process.

The [Steering Group] expects to grant [invitations][Workstream Participant Invitation] sparingly. 
Wherever possible, persons employed by, contracting, or otherwise working for an
[Entity] should seek to have the Entity execute the
[Contributor and Workstream Participant Agreement].

The [Steering Group] must approve an invitation by unanimous consent.

## How to request Workstream Participant Invitation status

1.	File an issue in the [Steering Group's repository](https://github.com/whatwg/sg/issues) requesting
    [Workstream Participant Invitation] status along with other relevant details or information 
    relevant to the request.
2.	The [Steering Group] will discuss the matter with the [Requestor] in the issue.
3.	Either the [Requestor] or the [Steering Group] may request a private meeting to discuss
    confidential special considerations.

> 4.  The scope of the [Requestor]'s [Workstream] participation is established.

Reviewer: Can we say something like "The Workstreams the Requestor will participate in are established. or does this mean something else? Perhaps this should also be explicit about the amount of time? That's also listed below, but perhaps that should only cover changes and this should cover the term itself?

Author: Sure, happy to re-phrase that to be more clear. I can also mention the 3 year term.

    
## Steering Group decision considerations

The [Steering Group] reserves the right to approve or reject a request for a 
[Workstream Participant Invitation] for any reason.

Some factors that the [Steering Group] will consider when evaluating a request 
are (but are not limited to):

* Hardship or difficulty in getting approval for a related [Entity] to join.
* The type and nature of the related [Entity] and potential IPR concerns.
*	The expected value of the potential participant’s [Contributions].
*	The degree of involvement to which the [Individual] or [Entity] is in the
  [Field of Web Technologies].
* Number of [Workstreams][Workstream] and desired participation length.
* Other specific circumstances related to the [Individual] or [Entity].

## Pre-approval

A person wishing to join the WHATWG using the [Workstream Participant Invitation] status 
must be pre-approved *before* attempting to sign the 
[Contributor and Workstream Participant Agreement]. Once pre-approval, 
as noted above, has been granted, the person may sign as an [Individual] with
[Workstream Participant Invitation] status. The [Requestor] will be added to the 

> [Workstreams][Workstream] that were previously negotiated with the [Steering Group].

Reviewer: I don't really care if people do things out-of-order and no harm will come to anyone when someone tries so I'm not sure it makes sense to say anything about it. You still need to be verified manually and that simply won't happen if there's no SG decision to be found to connect the signed agreement with.

Author: Yeah, I haven't liked this whole section since I came to understand how the system actually works. I'm considering replacing with a simpler explanation; something to the effect of: you still have to be validated regardless of signing the agreement.

Author: LMK what you think about the new section.

Reviewer: Seems good to me, thanks!

# WHATWG Workstream Participant Invitation Policy

## Joining the WHATWG via Workstream Participant Invitation

Persons who wish to join the WHATWG and participate in their personal capacity
(the <a id="requestor">**Requestor**</a>), but for various reasons are unable to sign the
[Contributor and Workstream Participant Agreement] as an [Entity] or as an [Individual] not
working in the [Field of Web Technologies] (for example, the [Individual] is working in the
[Field of Web Technologies] and the [Entity] employing the individual is unable to join),
may request special <a id="ws-invitation">**Workstream Participant Invitation**</a> status.
This policy describes the conditions and procedures for this process.

The [Steering Group] expects to grant [invitations][Workstream Participant Invitation] sparingly. 
Wherever possible, persons employed by, contracting, or otherwise working for an
[Entity] should seek to have the Entity execute the
[Contributor and Workstream Participant Agreement].

The [Steering Group] must approve an invitation by unanimous consent.

## How to request Workstream Participant Invitation status

> 1.	File an issue in the [Steering Group's repository](https://github.com/whatwg/sg/issues) requesting

Reviewer 2: ```suggestion
1.	File an issue in the [Steering Group’s repository](https://github.com/whatwg/sg/issues) requesting
```

    [Workstream Participant Invitation] status along with other relevant details or information 
    relevant to the request.
2.	The [Steering Group] will discuss the matter with the [Requestor] in the issue.
3.	Either the [Requestor] or the [Steering Group] may request a private meeting to discuss
    confidential special considerations.
4.  The [Workstreams][Workstream] the [Requestor] will participate in and the term of the 
    [invitation][Workstream Participant Invitation] (usually 36 months) are established.

> 5.	Each [Steering Group Member] must indicate support in the issue (for example by a "+1"

Reviewer 2: ```suggestion
5.	Each [Steering Group Member] must indicate support in the issue (for example by a “+1”
```

    or explicit indication of support).

> 6.	Once all [Steering Group Member]s have indicated support, and the scope of the [Requestor]'s

Reviewer 2: ```suggestion
6.	Once all [Steering Group Member]s have indicated support, and the scope of the [Requestor]’s
```
